sdks/python/container/boot.go

```diff
@@ -91,11 +92,14 @@ func main() {
 			"--container_executable=/opt/apache/beam/boot",
 		}
 		log.Printf("Starting worker pool %v: python %v", workerPoolId, strings.Join(args, " "))
-		if err := execx.Execute("python", args...); err != nil {
-			log.Fatalf("Python SDK worker pool exited with error: %v", err)
+		pythonVersion,err := expansionx.getPythonVersion()
+		if err == nil{
```

Comment:
I think a more common pattern is to handle error case, then proceed with normal flow:

```
if pythonVersion, err := expansionx.getPythonVersion(); err != nil {
    return err //  (or some other message + err)
}
<non-error flow>
```
any concerns with that ? thank you!